### ISSUE TITLE
Broken stereographic cursor coordinates

### ISSUE THREAD

**Issue by rhattersley** [SciTools/cartopy#141]
```python
import matplotlib.pyplot as plt
import cartopy
plt.axes(projection=cartopy.crs.Stereographic())
plt.show()
```

The lat-lon values for the cursor position always show 0'N 0'E.

The root cause is a bug in PROJ.4 where the inverse stereographic projection fails when equatorial and non-spherical. (See http://osgeo-org.1560.n6.nabble.com/Problem-with-inverse-stereo-projection-td3841805.html for a possible fix.)

Consider whether Stereographic should use WGS84 as the default ellipsoid. What would be the impact of a change to a spherical model?

**Comment by QuLogic** [comment]
[New ticket](https://trac.osgeo.org/proj/ticket/266), since the fix seems forgotten.

**Comment by QuLogic** [comment]
[New ticket from migration of proj to GitHub](https://github.com/OSGeo/proj.4/issues/266).

**Comment by QuLogic** [comment]
This is fixed in proj 4.9.2, but I'm going to leave this open as we're unsure whether to depend on 4.9+ just yet.

**Comment by pelson** [comment]
Closing, as this is a downstream issue, fixed in 4.9.2 of proj.4
